addons/base/views.py

```diff
@@ -281,100 +252,120 @@ def get_auth(auth, **kwargs):
         sentry.log_message(str(err))
         raise HTTPError(http_status.HTTP_403_FORBIDDEN)
 
-    if not auth.user:
-        auth.user = OSFUser.from_cookie(data.get('cookie', ''))
 
+def get_authenticated_node(node_id):
+    node = AbstractNode.load(node_id) or Preprint.load(node_id)
```

Comment:
nit: while we're here, can we stop universally using the term `node` in favor of `resource` or something else less domain-specific and more correct?